### DISCUSSION TITLE
Activating conpot_default on NextGen installation

### DISCUSSION THREAD

**Discussion by UncleRaymondo** [telekom-security/tpotce#1031] [Q&A]
M,

Just rebuilt the box here with your latest and greatest 😄 

Used the NextGen install. Is there anyway we can get conpot_default to turn on with it as well, as I don't see S7Comm, Modbus, or Bacnet ports appearing in dps.sh

I've got a client presentation next week and wouldn't mind showing them the dashboard with an attack or two on those protocols (alongside all the others!)

Cheers,
R

**Comment by t3chn0m4g3** [comment]
Yes, if you take care of the port conflicts for the other services you just need to copy and paste from `industrial.yml`.
You can test with `docker-compose -f nextgen.yml up` to see if any port conflicts are present.

Hope that helps 😄 

Cheers,
M.

**Reply by UncleRaymondo** [reply]
Cool - I've copied the Conpot Default from /opt/tpot/etc/compose/industrial.yml to /opt/tpot/etc/compose/nextgen.yml and commented out any conflicting ports (so that only s7comm, modbus and bacnet are running on this service)

![image](https://user-images.githubusercontent.com/5147133/156120339-5a24c4c9-869c-44f5-901d-f45f01571c6c.png)

![image](https://user-images.githubusercontent.com/5147133/156120274-c2075603-d17f-487e-8235-a9b42c65a846.png)

If I try `docker-compose -f nextgen.yml up` I now get this error

![image](https://user-images.githubusercontent.com/5147133/156120639-0fd55b4d-9fb7-425e-b867-a6b15b7605f5.png)

Running `docker network ls` shows the following

![image](https://user-images.githubusercontent.com/5147133/156120840-2a469bc2-d29d-4ae7-b886-875de3cb0696.png)

Not sure why these ones are appearing as compose_* and the original ones are etc_* - am I modifying an incorrect file? 

Any ideas what I'm doing wrong?

Cheers,
R

**Comment by UncleRaymondo** [comment]
Never mind M.... it would help if I actually stopped and restarted the tpot service before and after the changes *facepalm*

![image](https://user-images.githubusercontent.com/5147133/156157955-061a98ce-2ad1-4bee-bcb9-126c6d2c9230.png)


Going to try and split up that default one into 3 different services so it shows conpot_s7comm, conpot_modbus, conput_bacnet... 

Cheers,
R

**Comment by UncleRaymondo** [comment]
Ok the split worked :)

![image](https://user-images.githubusercontent.com/5147133/156159372-03f31b7d-f61d-4537-9594-c1e675379626.png)

now to try and attack the protocols and see what appears on the dashboard

Cheers,
R

**Reply by t3chn0m4g3** [reply]
Perfect, great it is working!

Cheers,
M.
